cmds/exp/tcpdump/gen.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	file, err := os.Create("well_known_ports.go")
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	fmt.Fprintln(file, `// Copyright 2024 the u-root Authors. All rights reserved
```

Comment:
as long as you're putting the whole string in memory, why not just use ioutil.WriteFile? Then it's just an easy single call and single error to check. You're not checking for an error in file.Close and that's not good.

Reply:
It now only uses ioutil.WriteFile